Dockerfile72

```diff
@@ -0,0 +1,9 @@
+MAINTAINER Lochemem Bruno Michael <lochbm@gmail.com>
```

Comment:
IMHO, these `Dockerfiles` should not be placed on the project root folder.

And it should have own folder to place them, maybe we can create a `.docker` directory to accomplish this issue.

Reply:
I'll take that under advisement.

Reply:
@ace411, I also think about creating Docker container environment.

If possible, I can create another repository named `docker_couch_ext` and it's a Docker image build repository.

And I think the Docker image build environment should be separated from `couchdb_ext` repository.

What do you think?